redi-connect-front/src/components/organisms/Carousel.scss

```diff
@@ -0,0 +1,41 @@
+@import "~bulma/sass/utilities/_all";
+@import "../../styles/_variables.scss";
+
+.slick-dots li {
+  width: .75rem;
+  height: .75rem;
+  background: $grey-light;
+  border-radius: 50%;
+
+  & button:before {
+    content: '';
+  }
+
+  &.slick-active {
+    background: $red;
+  }
+}
+
+.carousel {
+  &:focus {
+    outline: none;
+  }
+
+  &__image-border {
```

Comment:
A more optimal name for the element where you used this would be "__image". Since the element is not representing just a border. A border in this case is just an attribute of the image element.

Reply:
edited